build-support/bin/cherry_pick.sh

```diff
@@ -44,22 +55,25 @@ if [[ -z $CATEGORY_LABEL ]]; then
   read -r CATEGORY_LABEL
 fi
 REVIEWERS=$(gh pr view "$PR_NUM" --json reviews --jq '.reviews.[].author.login' | sort | uniq)
-BODY_FILE=$(mktemp "/tmp/github.cherrypick.$PR_NUM.$MILESTONE.XXXXXX")
-PR_CREATE_CMD=(gh pr create --base "$MILESTONE" --title "$TITLE (Cherry-pick of #$PR_NUM)" --label "$CATEGORY_LABEL" --body-file "$BODY_FILE")
-while IFS= read -r REVIEWER; do PR_CREATE_CMD+=(--reviewer "$REVIEWER"); done <<< "$REVIEWERS"
-BRANCH_NAME="cherry-pick-$PR_NUM-to-$MILESTONE"
 
-if [[ -z $COMMIT ]]; then
-  fail "Wasn't able to retrieve merge commit for $PR_NUM."
-fi
+for MILESTONE in $MILESTONES; do
+  BODY_FILE=$(mktemp "/tmp/github.cherrypick.$PR_NUM.$MILESTONE.XXXXXX")
```

Comment:
Minor, but the body file, `gh pr view` and `$COMMIT` check commands don't need to depend on the milestone anymore, since this now does all the milestones at once. For instance, this could be something like the following, and leave the loop body focused on just "create the branch and PR":

```shell
if [[ -z $COMMIT ]]; then
  fail "Wasn't able to retrieve merge commit for $PR_NUM."
fi

BODY_FILE=$(mktemp "/tmp/github.cherrypick.$PR_NUM.XXXXXX")
gh pr view "$PR_NUM" --json body --jq '.body' > "$BODY_FILE"

for MILESTONE in $MILESTONES; do
```